src/index.ts

```diff
@@ -176,6 +231,17 @@ export function instance<T = any>(options?: string | Options) {
 
 /**
  * Obtain metadata for the current GCP Project.
+ * @public
+ * @example
```

Comment:
@fhinkel I really love having the sample here in the docs, and also felt dirty writing it inline.  I know in other libraries we have example references here where it uses a jsdoc plugin @bcoe wrote to pull samples from disk at build time.  I suspect we're going to need to address this for launch.

Reply:
Also, it wasn't a bad dream, I also cannot get the examples to appear when I use the yaml generator.